### ISSUE TITLE
qDebug() prints "Got library name:" on every startup

### ISSUE THREAD

**Issue by dagheyman** [thp/pyotherside#113]
Every startup a message like this is printed to stdout, even for the release builds:
```
Got library name:  "/home/dag/Qt/5.12.4/gcc_64/qml/io/thp/pyotherside/libpyothersideplugin.so"`
```
Would you consider a patch defining [`QT_NO_DEBUG_OUTPUT`](https://doc.qt.io/qt-5/debug.html#warning-and-debugging-messages) for the release build?

**Comment by thp** [comment]
Yes, that'd be great if you could work on such a patch :)

**Comment by thp** [comment]
Fixed in PyOtherSide 1.6.0 by just removing the log message.
